wasm/src/lib.rs

```diff
@@ -89,25 +90,18 @@ pub fn get_library_source_content(name: &str) -> Option<String> {
 
 #[wasm_bindgen]
 pub fn get_hir(code: &str) -> String {
-    let (package, _) = compile(code);
-    package.to_string()
-}
-
-fn compile(code: &str) -> (qsc::hir::Package, Vec<VSDiagnostic>) {
-    STORE_CORE_STD.with(|(store, std)| {
-        let sources = SourceMap::new([("code".into(), code.into())], None);
-        let (unit, errors) = compile::compile(
+    let sources = SourceMap::new([("code".into(), code.into())], None);
+    let package = STORE_CORE_STD.with(|(store, std)| {
+        let (unit, _) = compile::compile(
```

Comment:
I'm curious, what was the reason for dropping errors here?

Reply:
The only (remaining) purpose of this function is to populate the HIR view in the playground . We don't need the compilation errors when formatting the HIR - they're already shown in the UI via the editor.

![image](https://github.com/microsoft/qsharp/assets/16928427/efe87816-118d-4180-8759-bf90421c1010)